src/vs/workbench/services/extensionManagement/browser/extensionBisect.ts

```diff
@@ -315,7 +314,7 @@ registerAction2(class extends Action2 {
 				await extensionEnablementService.disableExtension({ id: done.id }, undefined);
 			}
 			if (res.confirmed) {
-				await issueService.openReporter({ extensionId: done.id });
+				await commandService.executeCommand('OpenIssueReporterActionId', done.id);
 			}
```

Comment:
The ID is `workbench.action.openIssueReporter`

Reply:
completely slipped my mind.... 😭